website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,127 @@
+---
+page_title: Data Retention Policies - API Docs - Terraform Cloud
```

Comment:
Out of curiosity: Is it possible to remove "Terraform Cloud" from the page title here since this is a TFE-only feature, or is that page title breadcrumb necessary regardless?

Reply:
That is a good question! I am also wondering whether it makes most sense to add this page only to the enterprise section of the docs instead.

It may make sense to review the content here (so we have it in one place) but I can move this page to enterprise docs if we are happy with it